en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -15,12 +15,13 @@ We'll go through each of these in detail in the following sections.
 | Main interface | Function |
 | :--- | :--- |
 | Power | Connect Power module;  power input & AD voltage  and current detection. |
-| PM2 | Connect i2c smart battery. |
-| TF CARD | Insert SD card for log storage |
+| PM2 | [Do not use with PX4](../flight_controller/cuav_v5_nano.md#issue_pm2) |
```

Comment:
FYI only - This doc is a quick assembly guide for nano.  This fix just links to the issue about not using PM2 with PX4.

Reply:
@cuhome Can you please check this whole list for the nano? I think it was just copy-pasted, and it needs to be validated.